src/winforms/toga_winforms/app.py

```diff
@@ -80,5 +81,8 @@ def main_loop(self):
         thread.Start()
         thread.Join()
 
+    def exit_application(self, sender, event):
+        self.interface.exit()
+
```

Comment:
It took me a minute to catch on that `exit_application()` was a native handler - for naming consistency, it might be a good idea to call this method `on_exit_application()`, so that the `on_` prefix clearly identifies the method as a handler.

Reply:
@freakboy3742 thanks for your comments. However, I tested it and it didn't work out. See my comments above.

The additional changes with the exit_application() method makes it even worse.

With this exit_application(), my implementation for the get_value() of MultilineTextInput on Windows doesn't return anything. And I tested it that if I revert your last change back, the get_value() does work.
With this change, when I click the "X" button, the exit method is called twice when the App is closed, though that should not impact the function, but it is not proper.

Reply:
@obulat , with this change, it will make my App platform specific. Do you plan to make corresponding changes for cocoa and gtk?

Reply:
Actually, I only worked on Windows, but I tried implementing #522 on gtk on my Raspberry Pi, and it seemed to work. However, there were a couple of small stray errors there besides `on_exit` handling. Have you tried running your app on platforms other than Windows and Mac? Did it work?